src/main/java/frc/robot/Constants.java

```diff
@@ -141,6 +153,9 @@ public static class AimingLimelight {
        * Limelight's drive routine "done"
        */
       public static final double DISTANCE_DONE_THRESHOLD = 0;
+
+      /** The Limelight pipeline number for amp AprilTags. */
+      public static final double AMP_PIPELINE_NUMBER = 6;
```

Comment:
I also think this should be moved into its own class assuming there will be more than one (if not ignore me)